### PR TITLE
Fix time periods with DST and test

### PR DESCRIPTION
**Pull request by kukirokuk**
This change is [Reviewable](https://reviewable.io/reviews/openprocurement/openprocurement.auctions.dgf/79)

### CODE REVIEW

File: openprocurement/auctions/dgf/tests/tender.py [hunk 1]
```diff
@@ -1102,6 +1102,22 @@ def test_dateModified_auction(self):
...
+        # check if DST working with different time periods
+        for i in (10, 90, 180, 210):
+            self.initial_data.update({
+                "auctionPeriod": {
+                    "startDate": (now + timedelta(days=i)).isoformat(),
+                }})
+            response = self.app.post_json('/auctions', {'data': self.initial_data})
+            tender_end_date_after = parse_date(response.json['data']['tenderPeriod']['endDate']).time()
+            list_of_bools.append(tender_end_date_before != tender_end_date_after)
+        self.assertEqual(any(list_of_bools), True)
```
Comment (Leits) [inline openprocurement/auctions/dgf/tests/tender.py]: assertTrue

File: openprocurement/auctions/dgf/models.py [hunk 2]
```diff
@@ -347,9 +348,9 @@ def initialize(self):
         if not self.tenderPeriod:
             self.tenderPeriod = type(self).tenderPeriod.model_class()
         now = get_now()
-        self.tenderPeriod.startDate = self.enquiryPeriod.startDate = now
+        self.tenderPeriod.startDate = self.enquiryPeriod.startDate = now.astimezone(TZ)
         pause_between_periods = self.auctionPeriod.startDate - (self.auctionPeriod.startDate.replace(hour=20, minute=0, second=0, microsecond=0) - timedelta(days=1))
-        self.enquiryPeriod.endDate = self.tenderPeriod.endDate = calculate_business_date(self.auctionPeriod.startDate, -pause_between_periods, self)
+        self.enquiryPeriod.endDate = self.tenderPeriod.endDate = calculate_business_date(self.auctionPeriod.startDate.astimezone(UTC), -pause_between_periods, self).astimezone(TZ)
```
Comment (yarsanich) [inline openprocurement/auctions/dgf/models.py]: PEP 8

File: openprocurement/auctions/dgf/models.py [hunk 3]
```diff
@@ -347,9 +348,9 @@ def initialize(self):
         if not self.tenderPeriod:
             self.tenderPeriod = type(self).tenderPeriod.model_class()
         now = get_now()
-        self.tenderPeriod.startDate = self.enquiryPeriod.startDate = now
+        self.tenderPeriod.startDate = self.enquiryPeriod.startDate = now.astimezone(TZ)
```
Comment (yarsanich) [inline openprocurement/auctions/dgf/models.py]: PEP 8

File: openprocurement/auctions/dgf/models.py (hunk 2 above)
Comment (yarsanich) [inline openprocurement/auctions/dgf/models.py]: PEP 8

File: openprocurement/auctions/dgf/models.py (hunk 3 above)
Comment (yarsanich) [reply openprocurement/auctions/dgf/models.py]: Краще зробити окремо.

File: openprocurement/auctions/dgf/models.py (hunk 2 above)
Comment (yarsanich) [reply openprocurement/auctions/dgf/models.py]: Тут так само.

File: openprocurement/auctions/dgf/models.py [hunk 4]
```diff
@@ -1,5 +1,6 @@
 # -*- coding: utf-8 -*-
 from datetime import datetime, timedelta, time
+from pytz import UTC
```
Comment (yarsanich) [inline openprocurement/auctions/dgf/models.py]: PEP 8

**Pull request closed without merging**